### PR TITLE
add tf cnn back

### PR DESCRIPTION
**Pull request by mli**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
When was this removed earlier?

**Comment by mli** [comment]
Job d2l-en/PR-1882/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1882/

**Comment by mli** [comment]
Job d2l-en/PR-1882/8 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1882/

File: chapter_convolutional-modern/alexnet.md [hunk 1]
```diff
@@ -398,3 +435,7 @@ trainer.fit(model, data)
 :begin_tab:`pytorch`
 [Discussions](https://discuss.d2l.ai/t/76)
 :end_tab:
+
+:begin_tab:`pytorch`
```
Comment (AnirudhDagar) [inline chapter_convolutional-modern/alexnet.md:439]:
```suggestion
:begin_tab:`tensorflow`
```

This should be a tensorflow tab.

File: chapter_convolutional-modern/alexnet.md [hunk 2]
```diff
@@ -297,6 +297,12 @@ import torch
 from torch import nn
 ```
 
+```{.python .input}
+%%tab tensorflow
```
Comment (AnirudhDagar) [inline chapter_convolutional-modern/alexnet.md:301]:
```suggestion
#@tab tensorflow
```
Why is this not following this syntax? Looks confusing.

Comment (AnirudhDagar) [reply chapter_convolutional-modern/alexnet.md:301]: Similarly for places below.

**Review by AnirudhDagar — CHANGES_REQUESTED** [review]
@mli I've made some comments. PTAL!

File: chapter_convolutional-modern/alexnet.md (hunk 1 above)
Comment (mli) [reply chapter_convolutional-modern/alexnet.md]: good catch, thanks.

**Pull request merged**